### PR TITLE
Link Updates & Remove MediaElement sample

### PR DESCRIPTION
**Pull request by stmoy**
## Description

Added, removed, updated a bunch of links to align with the docs. Also removed the MediaElement sample.

## Motivation and Context

Aligns the Gallery with the docs.

## How Has This Been Tested?

Manual testing

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: XamlControlsGallery/DataModel/ControlInfoData.json [hunk 1]
```diff
@@ -1593,12 +1589,11 @@
...
             {
               "Title": "Media Playback",
-              "Uri": "http://windowsstyleguide/controls-and-patterns/media-playback/"
+              "Uri": "https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/media-playback"
             }
           ],
           "RelatedControls": [
-            "Image",
-            "MediaElement"
+            "Image"
```
Comment (YuliKl) [inline XamlControlsGallery/DataModel/ControlInfoData.json]: Would AnimatedVisualPlayer also be a related control? (Especially given that AVP doesn't have any related controls at all right now)

Comment (stmoy) [reply XamlControlsGallery/DataModel/ControlInfoData.json]: Maybe? Not really. AVP and MediaPlayerElement have little to nothing in common. Even Image is a stretch, I think.

**Comment by YuliKl** [comment]
Just noticed that ContentDialog's Guidelines link goes to https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/dialogs-and-flyouts/index. Should we link to https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/dialogs-and-flyouts/dialogs instead, seeing as that's the Gallery page the user came from?

**Comment by chigy** [comment]
> Just noticed that ContentDialog's Guidelines link goes to https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/dialogs-and-flyouts/index. Should we link to https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/dialogs-and-flyouts/dialogs instead, seeing as that's the Gallery page the user came from?

Yes, I agree @YuliKl

**Comment by stmoy** [comment]
> Just noticed that ContentDialog's Guidelines link goes to https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/dialogs-and-flyouts/index. Should we link to https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/dialogs-and-flyouts/dialogs instead, seeing as that's the Gallery page the user came from?

Good catch, updated.

**Pull request merged**